.github/workflows/teamcity-integration.yml

```
@@ -0,0 +1,339 @@
+            TRIGGER_AGENTS=true
+          fi
+
+          echo "trigger_dashboard=$TRIGGER_DASHBOARD" >> $GITHUB_OUTPUT
+          echo "trigger_backend=$TRIGGER_BACKEND" >> $GITHUB_OUTPUT
+          echo "trigger_mcp=$TRIGGER_MCP" >> $GITHUB_OUTPUT
+          echo "trigger_agents=$TRIGGER_AGENTS" >> $GITHUB_OUTPUT
+
+      - name: Trigger TeamCity Build
+        env:
+          TEAMCITY_TOKEN: ${{ secrets.TEAMCITY_API_TOKEN }}
+        run: |
```

actionlint (1.7.8)

"github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)